Chess/moveby.h

```diff
@@ -0,0 +1,15 @@
+#ifndef H_MOVEBY
+#define H_MOVEBY
+#include <vector>
+#include <pair>
+
+class MoveBy
```

Comment:
As vedea doua variante:
Fie un MoveBy sa reprezinte o singura mutare, fie sa fie cum e acum un vector de mutari, context in care ar trebui redenumit in MovesBy

Reply:
As merge parca pe varianta sa fie o singura mutare si prin cod sa avem vector<MoveBy>
Ce spui?

Reply:
Ok, facem cu MoveBy cu o singura mutare